### PR TITLE
Colors overview

### LINKED ISSUE #429: Visualize colors
## Are you reporting a bug or a feature?

 - [ ] Bug
 - [x] Feature

## Expected Behavior

Colors nicely visualized, including different color formats :)
https://design.axa.com/toolkit/v2/getting-started/colors.html

### PR DESCRIPTION
**Pull request by danyball**
Fixes #429 

# Done is when (DoD):
- [ ] I have added UI/Unit tests for my changes
- [ ] I added modified component properties to the typescript typings
- [ ] Design has been reviewed with (here name of reviewer)
- [x] My changes are accordingly to our [Best Practices](https://github.com/axa-ch/patterns-library/blob/develop/CONTRIBUTION.md#best-practices)
- [x] Old tests and eslint rule are not broken
- [x] My changes are well documented in the README and showcased in the stories 
- [x] Tested with IE

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas (focus on **WHY**)
- [x] I have made/updated the ChangeLog Section in the README

### CODE REVIEW

**Review by 01231 — COMMENTED** [review]
Have you seen the link in the story from andy? I like it and I think it would fit good into the pattern-library.
https://design.axa.com/toolkit/v2/getting-started/colors.html

**Comment by danyball** [comment]
> Have you seen the link in the story from andy? I like it and I think it would fit good into the pattern-library.
> https://design.axa.com/toolkit/v2/getting-started/colors.html

Yes thats nice. I am doing some replaces on "scss string" to color each line. For me it was too expansive to beautify it like andys link. 

New ticket or invest some more time for that in this PR?

**Comment by danyball** [comment]
Load scss dynamically and make it more beatifuly like noahs link,.

**Comment by danyball** [comment]
Load scss variables with webpack:
'to-string-loader' must be replaced by "style-loader. Problem.

**Comment by danyball** [comment]
PR ist soweit fertig. Dynamisches laden von SCSS Code als Text habe ich nicht hinbekommen - ist aber auch kein nogo würde ich sagen.

**Review by 01231 — CHANGES_REQUESTED** [review]
would be nice if all boxes are aligned
Screenshot 2020-01-22 at 09 29 06

File: src/components/00-materials/styles/story.js [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+
+const colors = `
+// Colors as specified at
+// @link https://design.axa.com/web-guidelines/color
+
+/// AXA brand colors
+$color-axa-blue: #00008f !default;
+$color-axa-red: #ff1721 !default;
+
+/// main palette colors (excluding brand colors)
```
Comment (01231) [inline src/components/00-materials/styles/story.js:22]: make all headings uppercase

File: src/components/00-materials/styles/story.js [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+$color-prim-gray-dusty: #999 !default;
+$color-prim-gray-silver: #ccc !default;
+$color-prim-gray-mercury: #e5e5e5 !default;
+$color-prim-gray-wild-sand: #f5f5f5 !default;
+$color-prim-gray-alabaster: #fafafa !default;
+
+$color-prim-black: #000 !default;
+$color-prim-white: #fff !default;
+
+/// secondary palette
```
Comment (01231) [inline src/components/00-materials/styles/story.js:43]: dito

File: src/components/00-materials/styles/story.js [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+
+/// Validation
+$color-validation-error: $color-sec-red-shy-tomato;
+$color-validation-valid: #1cc54e !default;
+
+//#########################################################################
+//################  The following colors are deprecated  ##################
+//#########################################################################
+
+/// main palette colors (excluding brand colors)
```
Comment (01231) [inline src/components/00-materials/styles/story.js:68]: dito

File: src/components/00-materials/styles/story.js [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+$color-prim-gray-mine-shaft: #333 !default; // deprecated name
+$color-prim-gray-scorpion: #5f5f5f !default; // deprecated name
+
+/// secondary palette
```
Comment (01231) [inline src/components/00-materials/styles/story.js:72]: dito

**Review by 01231 — CHANGES_REQUESTED** [review]

**Comment by raphaellueckl** [comment]
Screenshot 2020-01-22 at 09 29 18

Here is my review

**Comment by 01231** [comment]
@raphaellueckl as far as I have seen the <axa-text> doesn't work and we have to fix this bug first before we can use <axa-text> here

**Comment by danyball** [comment]
> @raphaellueckl as far as I have seen the doesn't work and we have to fix this bug first before we can use here

I put font-familiy to the body css. That works.

**Comment by raphaellueckl** [comment]
Hier noch ein paar Vorschläge:

```
.colorwrapper {
    ...
    display: flex;
    flex-direction: column;
    align-items: center;
}

.colorvisualisation {
margin-bottom: 10px;
}

body {
font-family: Source Sans Pro,Arial,sans-serif;
}


```

Führt zu folgendem Resultat:

Screenshot 2020-01-22 at 10 59 27

File: src/components/00-materials/styles/story.js [hunk 5]
```diff
@@ -0,0 +1,187 @@
...
+
+/*
+ * Just copy & paste 00-colors.scss to const colors below
+ * */
+
+const colors = `
+// Colors as specified at
+// @link https://design.axa.com/web-guidelines/color
+
+/// AXA brand colors
```
Comment (raphaellueckl) [inline src/components/00-materials/styles/story.js:18]: Three slashes?

**Review by raphaellueckl — APPROVED** [review]

**Review by 01231 — APPROVED** [review]

**Pull request merged**